examples/Cavity2D/CSMdummy.py

```diff
@@ -36,51 +32,41 @@ def computeDisplacements(force_vals, displ_vals, coords_x):
 
     return displ_vals
 
-
 solver_process_index = 0
 solver_process_size = 1
 
-interface = precice.Participant(participant_name, configuration_file_name, solver_process_index, solver_process_size)
+interface = precice.Participant(participant_name, configuration_file_name,
+                                 solver_process_index, solver_process_size)
+
 
 dim = interface.get_mesh_dimensions(mesh_name)
+print("Mesh dimensions:", dim)
 
 vertexSize = 100
 coords_x = np.linspace(0, 1, num=vertexSize)
 coords_y = np.linspace(0, 0, num=vertexSize)
-
 coords = np.stack([coords_x, coords_y], axis=1)
-
 print("coordinate array to be sent to set_mesh_vertices = {}".format(coords))
-print("mesh_name sent to set_mesh_vertices = {}".format(mesh_name))
+print("mesh_id sent to set_mesh_vertices = {}".format(mesh_name))
 
 vertexIDs = interface.set_mesh_vertices(mesh_name, coords)
-
-displ_name = "Displacements"
-force_name = "Forces"
 displacements = np.zeros([vertexSize, dim])
 forces = np.zeros([vertexSize, dim])
 solver_dt = 1.0
-if interface.requires_initial_data():
-    interface.write_data(mesh_name, force_name, vertexIDs, forces)
-    interface.write_data(mesh_name, displ_name, vertexIDs, displacements)
-
```

Comment:
Why did you remove this? Generally we should initialize data. If this does not work at the moment: Let's at least put a todo here.